package.json

```diff
@@ -137,6 +137,7 @@
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
     "@vscode/test-electron": "^2.1.3",
+    "@vscode/webview-ui-toolkit": "^0.9.3",
```

Comment:
I used `vscode webview-ui-toolkit` for vscode-specific tags in html.
- intro: https://code.visualstudio.com/blogs/2021/10/11/webview-ui-toolkit
- git: and this: https://microsoft.github.io/vscode-webview-ui-toolkit/?path=/story/library-badge--default